src/System.Net.Http/ref/System.Net.Http.csproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.1</AssemblyVersion>
```

Comment:
Yeah, desktop does need to match.  It looks like this library is reusing the single build of the ref for desktop and netstandard (most other libraries reuse their desktop implementation as the ref, since that targets mscorlib).   We can increment the refs in this case.